src/Aspire.Hosting.SqlServer/SqlServerBuilderExtensions.cs

```diff
@@ -101,25 +101,33 @@ public static IResourceBuilder<SqlServerServerResource> WithDataVolume(this IRes
     /// <param name="source">The source directory on the host to mount into the container.</param>
     /// <param name="isReadOnly">A flag that indicates if this is a read-only mount.</param>
     /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    /// <remarks>
+    /// The container starts up as non-root and the <paramref name="source"/> directory must be readable by the user that the container runs as.
+    /// https://learn.microsoft.com/sql/linux/sql-server-linux-docker-container-configure?view=sql-server-ver16&amp;pivots=cs1-bash#mount-a-host-directory-as-data-volume
+    /// </remarks>
     public static IResourceBuilder<SqlServerServerResource> WithDataBindMount(this IResourceBuilder<SqlServerServerResource> builder, string source, bool isReadOnly = false)
     {
         ArgumentNullException.ThrowIfNull(builder);
         ArgumentException.ThrowIfNullOrEmpty(source);
 
-        // c.f. https://learn.microsoft.com/sql/linux/sql-server-linux-docker-container-configure?view=sql-server-ver15&pivots=cs1-bash#mount-a-host-directory-as-data-volume
```

Comment:
@sebastienros - What do you think about not porting this change at all for release/9.1?

I think the risk here isn't worth it. Sql emulator in 9.1 currently "works" as it did in 9.0 right? So we are only adding risk here. I think we should keep this change targeted to the broken scenarios.